R/antigen.garnish_run.R

```diff
@@ -171,6 +171,135 @@ ag_data_err <- function(){
 
 }
 
+#' Internal function to configure netMHC suite of tools
+#'
+#' @param dir Character vector. Path to `antigen.garnish` data directory.
+#'
+#' @export configure_netMHC_tools
+#' @md
+
+configure_netMHC_tools <- function(dir){
+
+  owd <- getwd()
+
+  on.exit({setwd(owd)})
+
+  # netMHC parent dir in ag data dir
+  npd <- file.path(dir, "netMHC")
+
+  if (!dir.exists(npd)) dir.create(npd)
+
+  # get path to scripts
+  setwd(npd)
+
+  tool_paths <- list.files()
+
+  f <- lapply(tool_paths, function(i){
+
+    # this pattern could one day break.
+    list.files(i, pattern = "netMHC(II)?(pan)?-?([0-9]\\.[0-9])?$",
```

Comment:
make this case insensitive; extraction from archives does not preserve case on some systems

Reply:
is there a better way than just [Nn][Ee][Tt][Mm] etc? if not will do that.

Reply:
`(?i)`

Reply:
ignore.case = TRUE solves it